### PR TITLE
Spec fixes

### PR DESCRIPTION
**Pull request by dionyziz**

### CODE REVIEW

File: SPECIFICATION.md [hunk 1]
```diff
@@ -72,17 +72,15 @@ Each creature is associated with a **creatureid**, a unique positive integer tha
...
 
 ## Grader
 
-The game is overseen by a program called **grader**. The grader sets the game up, communicates with the bots, determines the configuration of each round, determines the outcome of the game, and tears down the game. Furthermore, the grader is responsible for enforcing game rules. As the grader is the de facto rules arbitrator, the source code of the grader is open source and can be used as a reference to the game rules. The de jure rules of the documentation are only incidental and derived from the grader’s source code, which is the normative source.
+The game is overseen by a program called the **grader**. The grader sets the game up, communicates with the bots, determines the configuration of each round, determines the outcome of the game, and tears down the game. Furthermore, the grader is responsible for enforcing game rules. As the grader is the *de facto* rules arbitrator, the source code of the grader is open source and can be used as a reference to the game rules. The *de jure* rules of the documentation are only incidental and derived from the grader’s source code, which is the normative source.
```
Comment (vitsalis) [inline SPECIFICATION.md:83]: _de facto_ ? why make that italic?

File: SPECIFICATION.md [hunk 2]
```diff
@@ -114,7 +112,7 @@ The grader shares the full state of the world with all players at the completion
 
 ## Intents
 
-Upon the completion of every round, and as long as there are more than two creatures belonging to different players alive, the grader informs each player about the current full state of the world and inquires each player about what they want to do. This phase is called the commit phase.
+Upon the completion of every round, and as long as there are more than two creatures belonging to different players alive, the grader informs each player about the current full state of the world and inquires each player about what they want to do. This phase is called the **commit phase**.
```
Comment (vitsalis) [inline SPECIFICATION.md:115]: here you make commit phase bold but on line 129 it's not.

File: SPECIFICATION.md [hunk 3]
```diff
@@ -242,7 +250,7 @@ The following general errors are logged and reported:
 
 The request may also contain JSON in the POST data. This is true for any non-scalar data structure sent from the grader to the bots. These pieces of JSON are again dictionaries, which contain keys specific to the particular request.
 
-Each bot must be aware of the username of the programmer which it is configured to play for. This user id is displayed in the bot configuration page, along with the api_key.
+Each bot must be aware of the username of the programmer which it is configured to play for. This helps ensure that the bot is playing for its rightful owner.
```
Comment (vitsalis) [inline SPECIFICATION.md]: whitespace on the end of the line

File: SPECIFICATION.md (hunk 2 above)
Comment (dionyziz) [inline SPECIFICATION.md:115]: Yes, I use bold at the locations where a term is first defined.

**Pull request merged**